spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
from now, all your code needs to have test coverage, and it has to be 100% alright?

Reply:
okay

Reply:
Okay. I tested it and it was 100%
I'm learning new ways to keep the code fully covered

Reply:
even the controllers?

Reply:
no, i will add

Reply:
Mark as resolved